src/sentry/utils/redis.py

```diff
@@ -53,27 +54,60 @@ def make_rb_cluster(*args, **kwargs):
     return _make_rb_cluster(*args, **kwargs)
 
 
+class _RBCluster(object):
+    def supports(self, config):
+        return not config.get('is_redis_cluster', False)
+
+    def factory(self, **config):
+        return _make_rb_cluster(**config)
+
+    def __str__(self):
+        return 'Redis Blaster Cluster'
+
+
+class _RedisCluster(object):
+    def supports(self, config):
+        return config.get('is_redis_cluster', False)
+
+    def factory(self, **config):
+        nodes = map(lambda n: n[1], config.get('hosts').items())
```

Comment:
> Isn't this the same as config.get('hosts').values()?

Yeah it totally is, surprised I missed that.

--

Are you saying in the `redis.clusters` configuration, it should just be formatted differently than how we're formatting the rb hosts?